src/coordinates.rs

```diff
@@ -0,0 +1,184 @@
+//! Custom data types used in the Gerber format.
+
+use std::convert::Into;
+use conv::TryFrom;
```

Comment:
This is an external crate, because `TryFrom` is not yet stabilizied: https://github.com/rust-lang/rust/issues/33417